### PR TITLE
add sessions info

### PR DESCRIPTION
**Pull request by FumiyaYoshida**
add sessions info because speakers have been confirmed.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -91,15 +91,15 @@ <h3>タイムテーブル</h3>
                                 </tr>
                                 <tr>
                                     <td>19:05〜19:50</td>
-                                    <td>セッション1: 未定</td>
+                                    <td>私のGitの使い方（いろふさん）</td>
```
Comment (238y) [inline index.html:94]: Please modify  '私' to 'わたし'.

**Review by 238y — CHANGES_REQUESTED** [review]
Thank you for your modifying!
Please check and modify.

File: index.html (hunk 1 above)
Comment (FumiyaYoshida) [reply index.html:94]: I modified.

**Review by FumiyaYoshida — COMMENTED** [review]
I modified a content of session.

**Pull request merged**